docs/widgetbook-cloud/overview.mdx

```diff
@@ -1,5 +1,12 @@
-# Widgetbook Cloud Overview
+# Widgetbook Cloud
 
-Widgetbook Cloud serves as a sophisticated collaboration platform tailored to the needs of Flutter frontend teams. By offering a seamless way for developers to share their Widgetbook with designers, project managers, clients, and other stakeholders, the platform streamlines the review process and fosters efficient communication across teams.
+<Info>
+  Widgetbook Cloud is currently in closed beta. If you are interested in
+  participating, please apply
+  [here](https://ic7614zmz3s.typeform.com/to/bqG82ECS).
+</Info>
 
-<Image src="/assets/WidgetbookCloud.png" zoom />
+Widgetbook Cloud has two main features that can help you improve your development workflow:
+
+1. **Hosting Widgetbook Builds**: You can share your Widgetbook builds with your team or the community by uploading them to Widgetbook Cloud.
+2. **Zero-configuration UI Regression Detector**: Widgetbook Cloud automatically detects UI regressions in your Widgetbook builds by comparing builds across different branches or commits, and notifies you about the changes.
```

Comment:
Even though Figma is not integrated in the smoothest way, it's still possible to compare Flutter vs. Code easier than without Widgetbook Cloud. Thus, I would include it in the docs:

3. Figma vs. Flutter Comparison: You can connect your Flutter widgets to their corresponding Figma designs. Thereby, you can verify if the developer meets the design requirements.

Reply:
I would delete "Zero-configuration" from the title and rather put it in the description. I like the term but it makes the title unnecessarily long

Reply:
As Reviews has two features, I would consider making it clear which value propositions belong to hosting and which to reviews.

Maybe we say:

**Hosting**
Hosting Widgetbook Cloud Builds:

**Reviews**
1. UI Regression Detector:
2. Figma vs. Flutter Comparison: